src/Fabulous.XamarinForms/Xamarin.Forms.Core.Attributes.fs

```diff
@@ -6,6 +6,15 @@ open Xamarin.Forms
 
 module Application =
     let MainPage = Attributes.defineWidget ViewNode.getViewNode "Application_MainPage" (fun target -> (target :?> Xamarin.Forms.Application).MainPage) (fun target value -> (target :?> Xamarin.Forms.Application).MainPage <- unbox value)
+    let Resources = Attributes.defineWidget ViewNode.getViewNode "Application_Resources" (fun target -> (target :?> Xamarin.Forms.Application).Resources) (fun target value -> (target :?> Xamarin.Forms.Application).Resources <- unbox value)
+    let UserAppTheme = Attributes.defineWidget ViewNode.getViewNode "Application_UserAppTheme" (fun target -> (target :?> Xamarin.Forms.Application).UserAppTheme) (fun target value -> (target :?> Xamarin.Forms.Application).UserAppTheme <- unbox value)
+    let ModalPopped = Attributes.defineEvent<ModalPoppedEventArgs> ViewNode.getViewNode "Application_ModalPopped" (fun target -> (target :?> Xamarin.Forms.Application).ModalPopped)
+    let ModalPopping = Attributes.defineEvent<ModalPoppingEventArgs> ViewNode.getViewNode "Application_ModalPopping" (fun target -> (target :?> Xamarin.Forms.Application).ModalPopping)
+    let ModalPushed = Attributes.defineEvent<ModalPushedEventArgs> ViewNode.getViewNode "Application_ModalPushed" (fun target -> (target :?> Xamarin.Forms.Application).ModalPushed)
+    let ModalPushing = Attributes.defineEvent<ModalPushingEventArgs> ViewNode.getViewNode "Application_ModalPushing" (fun target -> (target :?> Xamarin.Forms.Application).ModalPushing)
+    let PageAppearing = Attributes.defineEvent<Page> ViewNode.getViewNode "Application_PageAppearing" (fun target -> (target :?> Xamarin.Forms.Application).PageAppearing)
```

Comment:
Maybe we'll need to rethink events to avoid leaking the real control in the event handler

Reply:
I guess we can create a create a FabulousAppLication inherits from Application and then exposes a Events with custom EventsArgs .Similar to FabulousContentPage that already exists ?

Reply:
Hmm no, better not subclass all controls 😅
I was more thinking about another define function
e.g. `Attributes.defineEventWithCustomArg<WhateverWeWant> (fun target -> target.MyEvent) (fun eventArgs -> WhateverWeWant)`

That way, we would have `onMyEvent(fn: WhateverWeWant -> 'msg)`